main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"flag"
```

Comment:
Not your mess, but we should use pflag instead of flag. It's a straight drop in though, only need to change the import and it'll work correctly.